individualmentoring

```diff
@@ -0,0 +1,101 @@
+# How to individually mentor a struggling student
+
+Context:
+
+Many of our students struggle during the JavaScript phase. If you want to mentor them on an individual basis, the folowing approach
+can help you mentor the student in a structured and efficient way. We've cut it up into 3 steps.
+
+##1.Asses
+
+The first step in making a plan for the students you are mentoring would be to assess the level of the student. 
+
+In assessing the skill level, we look for possession of the skillset taught in our curriculum, with as the most important skillset : fundamental knowledge of JavaScript. (and therewith fundamental knowledge of programming)
+
+The following general topics must be understood well with regard to JavaScript:
+
+Basics:
+
```

Comment:
What about if focus on the steps/process of mentoring supporting a student instead of repeating the curriculum?